src/opf/ed.jl

```diff
@@ -306,6 +311,11 @@ function solve!(opf::OPFModel{EconomicDispatch})
     model.ext[:termination_info][:ptdf_iterations] = niter
     model.ext[:termination_info][:termination_status] = st
 
+    pg_ = value.(model[:pg])
```

Comment:
This will error if you have no primal solution (unlikely for LPs, but possible if the model turns out to be infeasible, or if numerical errors).

You can keep track of these values throughout the optimization. They are computed at each iteration, before checking for violations.

Reply:
Let's keep it like that for now, it will be fixed together with #94